### PR TITLE
Homework/2

### PR DESCRIPTION
**Pull request by Davydovychyurii**
homework 2

### CODE REVIEW

**Review by vakal237078 — APPROVED** [review]

File: HW2/Animation/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Animation</title>
+  <style>
+    @import "style.css";
+  </style>
+</head>
+<body>
+  <div class="wrapper">
+    <div class="cube">
+      <h1>Sugar</h1>
```
Comment (AgentPrus) [inline HW2/Animation/index.html:15]: maybe it should be Shugar?

File: HW2/Responsive/src/style.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  --thumbnail--description: #5d5d5d;
+  --btn-color: #fff;
+  --btn-bg-color: #5ebcff;
+  --navigation-border-bottom: #ddd;
+  --thumbnail-title: #484747;
+}
+
+body {
+  background-color: var(--bg-color--dark);
+  /* max-width: 991px; */
```
Comment (AgentPrus) [inline HW2/Responsive/src/style.css]: You needed this commented line of code ? If no, please remove this line of code

File: HW2/Animation/index.html (hunk 1 above)
Comment (Davydovychyurii) [reply HW2/Animation/index.html:15]: this is correct

File: HW2/Responsive/src/style.css (hunk 2 above)
Comment (Davydovychyurii) [reply HW2/Responsive/src/style.css]: +

**Pull request merged**